constants.go

```diff
@@ -337,6 +337,9 @@ const (
 	// Endpoint is an optional Host for non-AWS S3
 	Endpoint = "endpoint"
 
+	// UseFIPSEndpoint allows toggling FIPS endpoints for AWS S3/Dynamo
+	UseFIPSEndpoint = "use_fips_endpoint"
```

Comment:
IMO putting constants here is often an antipattern. You only ever use `UseFIPSEndpoint` in one package, so move it there and unexport it.

Reply:
@zmb3 Actually it is used in both `dynamoevents` and `s3sessions` - should I move them somewhere into the `events` package, leave them where they are, or duplicate them into each individual package?